python/lsst/ts/genericcamera/driver/alliedvisioncamera.py

```diff
@@ -171,6 +174,7 @@ def get_roi(self):
             The height of the region in pixels.
         """
         with vimba.Vimba.get_instance():
+            # self.camera = v.get_camera_by_id(self.id)
```

Comment:
Is it absolutely necessary to keep this commented out line? If not, please remove.

Reply:
Removed.